<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexViewerForm</class>
 <widget class="QWidget" name="HexViewerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sumuri.qrc">
    <normaloff>:/icons/app_logo/hex_viewer_icon.ico</normaloff>:/icons/app_logo/hex_viewer_icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Bytes per Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bytesPerLinecomboBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="sumuri.qrc">
         <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchNextButton">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="go-next"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="jumpToOffsetButton">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="sumuri.qrc">
         <normaloff>:/icons/jump.png</normaloff>:/icons/jump.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showTablesButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Show Tables</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="readAsImageButton">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Read as image </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" alignment="Qt::AlignTop">
    <widget class="QTabWidget" name="tagstabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>204</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tagsTab">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tags&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <attribute name="title">
       <string>Tags</string>
      </attribute>
      <widget class="QTableView" name="tagsTableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>871</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>363</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="removeTagButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportTagDataButton">
          <property name="text">
           <string>Export Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportTagsButton">
          <property name="text">
           <string>Export Tags</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="importTagsButton">
          <property name="text">
           <string>Import Tags</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="templateTagsTab">
      <attribute name="title">
       <string>Template Tags</string>
      </attribute>
      <widget class="QTableView" name="TemplateTagstableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>691</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>647</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="removeTemplateTagButton">
          <property name="text">
           <string>Remove Template Tags</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportTemplateTagButton">
          <property name="text">
           <string>Export Template  Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportTemplateTagsButton">
          <property name="text">
           <string>Export Template Tags</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="importTemplateTagsButton">
          <property name="text">
           <string>Import Template Tags</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="MarkersTab">
      <attribute name="title">
       <string>Markers</string>
      </attribute>
      <widget class="QTableView" name="markersTableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>871</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="fileSystemTab">
      <attribute name="title">
       <string>File System</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="HexEditor" name="hexEditorWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexEditor</class>
   <extends>QWidget</extends>
   <header>headers/hexeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sumuri.qrc"/>
 </resources>
 <connections/>
</ui>
